response-to-reviewers.md

```diff
@@ -92,35 +99,28 @@ and fully agree that the language in the manuscript needs to better emphasize th
 To address this, we have made several changes throughout the manuscript, including the passages cited above:
 
 The first quoted passage now reads:
-"For each organism, the catalog contains information on our current understanding of
-the physical organization (e.g., chromosome structure) of its genome,
-one or more inferred genetic maps (except for \textit{E.~coli}),
-default population-level parameters (mutation rate and generation time estimates).
-The catalog also contains numerous published demographic models for the first three species.
-These available models and parameters are meant to represent the field's current understanding,
-and we intend for this resource to evolve as new results become available."
+"For each species, the catalog contains curated information on our current understanding of
+the physical organization of its genome, inferred genetic maps,
+population-level parameters (e.g., mutation rate and generation time estimates),
+and published demographic models.
+These models and parameters are meant to represent the field's current understanding,
+and we intend for this resource to evolve as new results become available, and
+other existing models are added to stdpopsim by the community."
```

Comment:
Throughout, I've checked what we actually say in the manuscript and updated this.